HACKING.md

```diff
@@ -1,5 +1,7 @@
 # Snooty Parser
 
+![Data flowchart](./docs/flowchart.svg)
```

Comment:
(Non-blocking) This flowchart talks about adding the AST to the db, but that's now handled by the persistence module. We should consider updating a bit when we have the time!

Reply:
Updated!